packages/cli/command_spec.pony

```diff
@@ -79,12 +80,13 @@ class CommandSpec
     if _args.size() > 0 then error end
     _commands.update(cmd.name(), cmd)
 
-  fun ref add_help(hname: String = "help") ? =>
+  fun ref add_help(hname: String = "help", hinfo: String = "Print help page") ? =>
```

Comment:
Maybe i should leave this default to empty

Reply:
i'd suggest to rename `hinfo` to `descr'` to accomodate to the API of [`OptionSpec`](https://stdlib.ponylang.org/cli-OptionSpec/) and maybe keep a default one of `display this help and exit`.

Reply:
It should default to empty for this to be considered a "non-breaking change".